### PR TITLE
Branch user guide

### PR DESCRIPTION
**Pull request by Shannonwje**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -7,14 +7,15 @@
 * **[3. Features](#features)**
   * [3.1 Adding an item: `ADD`](#adding-an-item-add)
   * [3.2 Editing an item: `EDIT`](#editing-an-item-edit)
-  * [3.3 Deleting an item: `Delete`](#deleting-an-item-del)
+  * [3.3 Deleting an item: `DEL`](#deleting-an-item-del)
```
Comment (JLoh579) [inline docs/UserGuide.md:10]: Thanks for spotting the error! I edited this line as well, but I realised I renamed to DELETE instead, my bad

File: docs/UserGuide.md [hunk 2]
```diff
@@ -7,14 +7,15 @@
...
   * [3.7 Setting a budget: `SET`](#setting-a-budget-set)
-  * [3.8 Finding an item: `FIND`](#finding-an-item-find)
-  * [3.9 Viewing help: `HELP`](#viewing-help-help)
-  * [3.10 Exiting the Program: `BYE`](#exiting-the-program-bye)
+  * [3.8 Resetting a budget: `RES`](#resetting-a-budget-res)
+  * [3.9 Finding an item: `FIND`](#finding-an-item-find)
+  * [3.10 Viewing help: `HELP`](#viewing-help-help)
+  * [3.11 Exiting the Program: `BYE`](#exiting-the-program-bye)
 * **4. [FAQ](#faq)**
 * **5. [Command Summary](#command-summary)**
```
Comment (JLoh579) [inline docs/UserGuide.md]: I changed these lines in the PR I merged to make them consistent with the lines above. If it shows up in the merge conflict can help me keep those changes? Thanks!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -184,6 +201,21 @@ Example of usage:
...
 ***
 
+### Resetting a budget: `RES`
+Resets te budget to be 0.00 for the user.
+
+Format: `RES`
+
+Example of usage: 
+
+`RES`
```
Comment (JLoh579) [inline docs/UserGuide.md]: I think for the simple commands we aren't including an example of usage, since its the same as in the format

**Review by JLoh579 — COMMENTED** [review]
Hey Shannon, there's some merge conflicts, so I included some comments that might help in choosing what to keep!

File: docs/UserGuide.md (hunk 3 above)
Comment (Shannonwje) [reply docs/UserGuide.md]: Okiesz, will make the changes! Thank you :)

File: docs/UserGuide.md (hunk 1 above)
Comment (Shannonwje) [reply docs/UserGuide.md:10]: No probbies!! Changed it already. Thanks:)

File: docs/UserGuide.md (hunk 2 above)
Comment (Shannonwje) [reply docs/UserGuide.md]: Yep noted, will keep your changes! Thanks :)

**Pull request merged**